src/Simplex/Chat.hs

```diff
@@ -4331,11 +4336,17 @@ processAgentMessageConn vr user@User {userId} corrId agentConnId agentMessage =
       OK ->
         -- [async agent commands] continuation on receiving OK
         when (corrId /= "") $ withCompletedCommand conn agentMsg $ \_cmdData -> pure ()
+      QCONT -> do
+        processConnQCONT connEntity conn
+        lastSharedMsgId <- withStore' $ \db -> getLastItemSharedMsgId db m
```

Comment:
- for super peer it may be better to instead search via `msg_deliveries` table, to account for forwarded group messages (see `forwardMsg_`)
  - may have to additionally store original shared msg id (currently shared_msg_id is for the forwarded message itself)
- or populate `group_snd_item_statuses`?
- store additional data